js/main.js

```diff
@@ -10,6 +10,9 @@ Site = {
       _this.onResize();
     });
 
+    if ($('body').hasClass('post-type-archive-press')) {
```

Comment:
This isn't really needed since `if ($('.swiper-slide').length)` will be checked later. But it will really depend if `Site.Press` gets renamed.